tsconfig.json

```diff
@@ -1,22 +1,29 @@
 {
   "compilerOptions": {
-    "preserveConstEnums": true,
-    "sourceMap": true,
     "target": "ES2018",
     "module": "esnext",
     "lib": [
       "dom",
       "es6",
       "es2015.promise"
     ],
-    "moduleResolution": "node",
-    "esModuleInterop": true,
-    "downlevelIteration": true,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "outDir": "lib",
     "strict": true,
-    "resolveJsonModule": true
+    "noImplicitThis": false,
```

Comment:
This is false by default:
https://www.typescriptlang.org/tsconfig#noImplicitThis

Reply:
```suggestion
```